### PR TITLE
[RI2][pcac]: Correcting document title

### PR DESCRIPTION
**Pull request by CsatariGergely**
Based on the discussion in #3045

### CODE REVIEW

File: doc/ref_impl/cntt-ri2/index.rst [hunk 1]
```diff
@@ -1,7 +1,8 @@
-Reference Implementation 2 - Kubernetes
-=======================================
+==========================================================
+Reference Implementation based on RA2 specifications (RI2)
+==========================================================
 
-This is the Kubernetes-based Reference Implementation (RI-2).
+This is the Kubernetes-based Reference Implementation (RI2).
```
Comment (pgoyal01) [inline doc/ref_impl/cntt-ri2/index.rst:5]: I don't think it is just Kubernetes based but RA2 based
```suggestion
This is the Reference Implementation (RI2) for the Kubernetes-based Reference Architecture, RA2.
```

**Review by gkunz — APPROVED** [review]
I am a little torn between the technically correct title "based on RA1 specs" and not having the target platform (OpenStack) in the title. But I am ok with the proposal.

**Comment by CsatariGergely** [comment]
> I am a little torn between the technically correct title "based on RA2 specs" and not having the target platform (Kubernetes) in the title. But I am ok with the proposal.

This is why I added the "This is ..." sentence after the title

**Review by pgoyal01 — APPROVED** [review]

**Review by rihabbanday — APPROVED** [review]

**Review by karinesevilla — APPROVED** [review]

**Review by steelescotr — APPROVED** [review]
Approved

**Pull request merged**